### PR TITLE
[dv] Clean up for style lint

### PR DESCRIPTION
**Pull request by weicaiyang**
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

**Comment by weicaiyang** [comment]
@msfschaffner 
I cleaned up all style lint error in DV, except
1. Syntax error within SV macros due to [tool issue](https://github.com/google/verible/issues/102)
2. rv_plic/dv. I'm not sure if you still want to maintain this TB, but some files are missing and it can't be run now.

File: hw/dv/sv/sw_msg_monitor_if/sw_msg_monitor_if.sv [hunk 1]
```diff
@@ -435,7 +435,7 @@ interface sw_msg_monitor_if #(
 
     case (msg_type.tolower())
       "i", "info":            `uvm_info(msg_header, msg, level)
-      "w", "warn", "warning": `uvm_warning(msg_header, msg)
+      "w", "warn", "warning": `uvm_error(msg_header, msg)
```
Comment (sriyerg) [inline hw/dv/sv/sw_msg_monitor_if/sw_msg_monitor_if.sv]: Why this change?

**Comment by weicaiyang** [comment]
@rasmus-madsen 
I updated AES DV to fix below style errors:
1. trailing spaces (no space at the end of the line)
2. Line length exceeds max (100 characters)
3. Explicitly define a storage type for every parameter and localparam

File: hw/dv/sv/sw_msg_monitor_if/sw_msg_monitor_if.sv (hunk 1 above)
Comment (weicaiyang) [inline hw/dv/sv/sw_msg_monitor_if/sw_msg_monitor_if.sv]: @sriyerg 
verilog_lint doesn't allow to use uvm_warning and uvm_warning also makes our simulation fail. So, I replace it with uvm_error.

**Comment by msfschaffner** [comment]
> @msfschaffner
> I cleaned up all style lint error in DV, except
> 
> 1. Syntax error within SV macros due to [tool issue](https://github.com/google/verible/issues/102)
> 2. rv_plic/dv. I'm not sure if you still want to maintain this TB, but some files are missing and it can't be run now.

That is maybe a better question for @eunchan or @cindychip. If it is not functional, should we update it or remove it? I guess this is something that would belong into `scratch_dv` (or whathever the name will be...)

**Comment by rasmus-madsen** [comment]
@weicaiyang 
thank you I will rebase once these get checked in.

I must say I find the limitations on linewidth to small.. I for one are strongly againts these wrapping lines. can you point me where this was decided I would like to understand the reason behind it :D

**Comment by weicaiyang** [comment]
> @weicaiyang
> thank you I will rebase once these get checked in.
> 
> I must say I find the limitations on linewidth to small.. I for one are strongly againts these wrapping lines. can you point me where this was decided I would like to understand the reason behind it :D

@rasmus-madsen
Here is the link of the rule.
https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#line-length
I feel 100 char limit is quite useful when you diff 2 files and review code changes, the line won't be auto-wrapped in most of the monitors. And we also have some macros (like`gfn, `gn for common functions - get_full_name, get_name), to avoid too much line wrapping.
I personally add `set colorcolumn=100` in .vimrc to know where to wrap the code in editor.

File: hw/ip/trial1/dv/trial1_test.sv [hunk 2]
```diff
@@ -21,9 +21,9 @@ module trial1_test (
   // for now always accept read responses
   assign  tl_h2d.d_ready = 1'b1;
 
-  task send_wr (
-    input [11:0] waddr,
-    input [31:0] wdata
+  task automatic send_wr (
+    input bit[11:0] waddr,
```
Comment (sriyerg) [inline hw/ip/trial1/dv/trial1_test.sv]: nit: space between bit and '[' (here and below)

Comment (weicaiyang) [reply hw/ip/trial1/dv/trial1_test.sv]: Done
